File: VNC.Domain/ViewModel/UserViewModel.cs
using MongoDB.Bson;
using System;

namespace VNC.Domain.ViewModel
{
    public class UserViewModel
    {
        public ObjectId Id { get; set; }
        public string IPAddress { get; set; }
        public string EmailAddress { get; set; }
        public DateTime ExpiresIn { get; set; }
        public string InExuer { get; set; }
        public string Gender { get; set; }
        public decimal LimitedTotal { get; set; }
    }
}

File: VNC.WebAPI/ClientApp/src/app/_models/user.viewmodel.ts
import { RoleDetailViewModel } from './roleDetails';

export class UserViewModel {

    constructor() {
      this.id = '';
      this.userName = '';
      this.password = '';
      this.emailAddress = '';
      this.gender = '';
      this.expiresIn = new Date;
      this.ipAddress = '';
      this.inExuer = '';
      this.roleDetails = new Array<RoleDetailViewModel>();
    }
  
    public id: string;
    public userName: string;
    public password: string;
    public emailAddress: string;
    public gender: String;
    public expiresIn: Date;
    public ipAddress: string;
    public createdDate: Date;
    public limitedTotal: number;
    public inExuer: string;
    public accessToken:string;
    public roleDetails: Array<RoleDetailViewModel>;
  }